manifests/osfamily/solaris.pp

```diff
@@ -52,7 +52,7 @@
         owner  => 0,
         group  => 0,
         mode   => '0644',
-        source => "puppet:///pe_packages/${pe_server_version}/${::platform_tag}/solaris-noask",
+        source => "puppet:///modules/pe_packages/${pe_server_version}/${::platform_tag}/solaris-noask",
```

Comment:
This is the path from where files are served by the server node and it will break functionality